### PR TITLE
Add deadline feature in user guide and change project name

### PR DESCRIPTION
**Pull request by tehkokhoe**

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -142,6 +142,19 @@ Examples:
...
+
+Format : `deadline INDEX DATE`
+
+- The given date is added to the client’s contact as deadline.
+- Date should be dd/mm/yyyy
+
+Example:
+
+- `deadline 1 01/01/2022` adds the date `01/01/2022` to the client contact in index `1`.
+
```
Comment (glennljw) [inline docs/UserGuide.md:157]: Perhaps you could also include this feature/command in the **command summary** section at the bottom of the User Guide.

**Review by glennljw — COMMENTED** [review]
LGTM 👍

File: docs/UserGuide.md (hunk 1 above)
Comment (tehkokhoe) [reply docs/UserGuide.md:157]: Okay.

File: docs/UserGuide.md [hunk 2]
```diff
@@ -142,6 +142,19 @@ Examples:
...
+Creates a deadline that is placed under the profile of a client.
+
+Format : `deadline INDEX DATE`
+
+- The given date is added to the client’s contact as deadline.
+- Date should be dd/mm/yyyy
+
+Example:
+
+- `deadline 1 01/01/2022` adds the date `01/01/2022` to the client contact in index `1`.
```
Comment (glennljw) [inline docs/UserGuide.md:156]: @weijuey Please help to do a quick review on the user entry consistency, not sure if "clients" and "contacts" should be mixed together 🙇‍♂️

Comment (weijuey) [reply docs/UserGuide.md:156]: I think this functionality can be applied to any contact in the list. Clients are just one type of contact, and the functionality should be useful for other types like leads and contractors. No worries though, as @takufunkai suggested, we'll open an issue just for standardising the term to contacts in the future. 👋

**Review by weijuey — APPROVED** [review]
Lgtm, no other problems, we'll settle clients and contacts on a different day 👍

**Pull request merged**